### DISCUSSION TITLE
Front-End capacitors

### DISCUSSION THREAD

**Discussion by William04567** [fhdm-dev/scoppy#55] [Q&A]
I am making this [this](https://oscilloscope.fhdm.xyz/wiki/front-end-design-1) version of the frond end. I didnt have the same capacitors but does it matter if I use larger values?

**Comment by fhdm-dev** [comment]
Hi,

The exact values of these capacitors don't matter. Larger values should be fine and for C2, the larger the better in order to keep Vref as stable as possible.

**Reply by William04567** [reply]
Thank you, 
Can I use 2 0.47uf electrolytic capacitors?

**Reply by fhdm-dev** [reply]
Yes. Definitely :smiley:
